easygoogle/src/main/java/pub/devrel/easygoogle/Google.java

```diff
@@ -72,6 +73,18 @@ public Builder enableSignIn(SignIn.SignInListener signInListener) {
         }
 
         /**
+         * Initialize {@link SignIn}.
+         * @param signInListener listener for sign in events.
+         * @param serverClientId client id for backend OAuth.
```

Comment:
Add in comments that this is optional, and also specify that it's for an ID token.  Suggestion:
`@param serverClientId (optional) web client id for requesting ID Token.`